app.js

```
@@ -124,315 +129,1806 @@ function getRiskLevel(points) {
+            },
+          },
+        ],
+      },
+    });
+  } catch (error) {
+    logger.error({ err: error }, "Failed to show success modal");
+  }
+});
+
+// Handle modal close (user clicked X, pressed Escape, or switched away)
+slackApp.view("carrier_wizard", async ({ ack, body, view }) => {
```

ESLint

[error] 'body' is defined but never used.

(no-unused-vars)